stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -2548,9 +2548,29 @@ impl NakamotoChainState {
             &parent_header_hash,
             &MINER_BLOCK_CONSENSUS_HASH,
             &MINER_BLOCK_HEADER_HASH,
-            new_tenure,
         );
 
+        if new_tenure {
```

Comment:
Do you have tests to verify that the tenure height gets initialized to `(+ u1 block-height)` for the first Nakamoto block?

Reply:
Are you thinking an integration test for this or some kind of unit test?

Reply:
Added to my integration test in 4fb8bdd86cbe108e2f0406ee52cc1d4879b2eb2e.